BlockServer/runcontrol/runcontrol_manager.py

```diff
@@ -316,31 +315,20 @@ def wait_for_ioc_start(self, time_between_tries=2):
 
         """
         print_and_log("Waiting for runcontrol IOC to start ...")
-        started = False
-        loop_count = 0
-        while not started and loop_count < MAX_LOOPS_TO_WAIT_FOR_START:
-            loop_count += 1
-            # See if the IOC has restarted
-            try:
-                if ioc_restart_pending(self._prefix + RC_IOC_PREFIX,
-                                       self._channel_access):
-                    raise Exception()
-
-                latest_ioc_start = self._get_latest_ioc_start()
 
-                if self._invalid_ioc_start_time(latest_ioc_start):
-                    raise Exception()
+        for loop_count in range(MAX_LOOPS_TO_WAIT_FOR_START):
+            restart_pending = ioc_restart_pending(self._prefix + RC_IOC_PREFIX,
+                                                  self._channel_access)
+            latest_ioc_start = self._get_latest_ioc_start()
 
+            if restart_pending or self._invalid_ioc_start_time(latest_ioc_start):
+                self._sleep_func(time_between_tries)
+            else:
                 self._rc_ioc_start_time = latest_ioc_start
-                started = True
                 print_and_log("... Runcontrol IOC started")
-            except Exception:
-                self._sleep_func(time_between_tries)
-        if not started:
-            print_and_log("Runcontrol appears not to have started", "MAJOR")
+                break
```

Comment:
The line `self._sleep_func(time_between_tries * 3)` has disappeared. I think it should go here.